content/20.pharmaceuticals.md

```diff
@@ -243,7 +244,23 @@ Patients who received HCQ also had a longer duration of hospitalization than pat
+Ivermectin has been used since the early 1980s to treat endo- and ecto-parasitic infections by helminths, insects, and arachnids in veterinary contexts [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.372] and since the late 1980s to treat human parasitic infections as well [@doi:10.1038/nrmicro1048; @doi:10.2183/pjab.87.13].
+More recent research has indicated that ivermectin might function as a broad-spectrum antiviral by distrupting the trafficking of viral proteins by both RNA and DNA viruses [@doi:10.1038/s41429-020-0336-z; @doi:10.1038/ja.2017.11; @doi:10.1186/s12941-020-00368-w], although most of these studies have demonstrated this effect _in vitro_ [@doi:10.1186/s12941-020-00368-w].
+The potential for antiviral effects on SARS-CoV-2 were investigated _in vitro_, and ivermectin was found to inhibit viral replication in a cell line derived from Vero cells (Vero-hSLAM) [@doi:10.1016/j.antiviral.2020.104787] but at a concentration that is much higher than that explored by existing dosage guidelines [@doi:10.4269/ajtmh.20-0271; @doi:10.1002/cpt.1889] and likely to be associated with significant side effects due to the increased potential that the compound could cross the mammalian blood-brain barrier [@doi:10.1080/13102818.2020.1775118; @doi:10.1371/journal.pntd.0001883].
+
+Retrospective studies and small RCTs began investigating the effects of standard doses of this low-cost, widely available drug.
+One retrospective study reported that ivermectin reduced all-cause mortality [@doi:0.1016/j.chest.2020.10.009] while another reported no difference in clinical outcomes or viral clearance [@doi:10.1371/journal.pone.0242184].
+Small RCTs enrolling less than 50 patients per arm have also reported a wide array of positive [@doi:10.4103/1995-7645.318304; @doi:10.1093/qjmed/hcab035; @doi:10.1016/j.toxrep.2021.03.003; @doi:10.1016/j.ijid.2020.11.191; @doi:10.1016/j.clinthera.2021.04.007] and negative results [@doi:10.1016/j.eclinm.2020.100720; @doi:10.21203/rs.3.rs-191648].
+A slightly larger RCT enrolling 115 patients in two arms reported inconclusive results [@doi:10.18433/jpps32105].
+Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:https://doi.org/10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].
```

Comment:
```suggestion
Hope for the potential of ivermectin peaked with the release of a preprint reporting results of a multicenter, double-blind RCT where a four-day course of ivermectin was associated with clinical improvement and earlier viral clearance in 400 symptomatic patients and 200 close contacts [@doi:10.21203/rs.3.rs-100956/v3]; however, concerns were raised about both the integrity of the data and the paper itself [@url:https://grftr.news/why-was-a-major-study-on-ivermectin-for-covid-19-just-retracted; @url:https://steamtraen.blogspot.com/2021/07/Some-problems-with-the-data-from-a-Covid-study.html], and this study was removed by the preprint server Research Square [@doi:10.21203/rs.3.rs-100956/v4].
```

I wanted to check how the retraction would render.  The authors aren't there, which makes sense, and it looks reasonable to me:
1. **Efficacy and Safety of Ivermectin for Treatment and prophylaxis of COVID-19 Pandemic**
Research Square Platform LLC *Research Square Platform LLC* (2020-10-30) <https://doi.org/gmhmfm>
DOI: [10.21203/rs.3.rs-100956/v4](https://doi.org/10.21203/rs.3.rs-100956/v4)